### ISSUE TITLE
Different output on python and C++

### ISSUE THREAD

**Issue by app-houqiangli** [NVIDIA/TensorRT#4305]
Hello, I have a model where both the input and output are float32. After providing the same input, I performed TensorRT inference on the same GPU (4090) using both the C++ and Python versions, and I obtained different floating-point outputs. The discrepancy starts from the third decimal place. Could you please help explain the reason for this? I look forward to your reply.
TensorRT version: 8.5.1.7
GPU:4090

**Comment by lix19937** [comment]
You should make sure the c++ and py version of trt are the same.  c++ map to libnvinfer.so, py map to tensorrt.so.

**Comment by asfiyab-nvidia** [comment]
@app-houqiangli Yes, as @lix19937 mentioned make sure the C++ and Python TRT versions installed in your env are the same. If you're still noticing a discrepancy, please provide additional information and scripts to reproduce this issue

**Comment by LeoZDong** [comment]
Closing as stale.
